superset/views/datasource/views.py

```diff
@@ -203,3 +203,27 @@ def samples(self) -> FlaskResponse:
             payload=payload,
         )
         return self.json_response({"result": rv})
+
+
+class DatasetEditor(BaseSupersetView):
+    route_base = "/dataset"
+    class_permission_name = "Dataset"
+
+    @expose("/add/")
+    @has_access
+    @permission_name("read")
```

Comment:
what sort of permissions does this map to? Who has permission to read this page?